io/disk/softwareraid.py

```diff
@@ -41,30 +41,33 @@ def setUp(self):
         Checking if the required packages are installed,
         if not found packages will be installed.
         """
+        self.disks = []
+        self.spare_disks = []
         smm = SoftwareManager()
         if not smm.check_installed("mdadm"):
             self.log.info("Mdadm must be installed before continuing the test")
             if SoftwareManager().install("mdadm") is False:
                 self.cancel("Unable to install mdadm")
-        disks = self.params.get('disks', default='').strip(" ")
+        disks = self.params.get('disks', default='').split()
```

Comment:
I think you have to add strip too as some time chances of adding space at the end.
please re-think on this